docs/src/pages/docs/functions/helpers.md

```diff
@@ -283,6 +283,66 @@ that can be called in any combination, such as: `f(x, y, z)`, `f(x)(y)(z)`,
 `f(x, y)(z)`, or even `f(x)(y, z)`. This is great for doing partial application
 on functions for maximum re-usability.
 
+```javascript
+import curry from 'crocks/helpers/curry'
+
+// add :: (Number, Number, Number) -> Number
+const add = (a, b, c) =>
+  a + b + c
+
+// partial application impossible
+add(1)
+// => 1 + undefined + undefined => NaN
+
+// NOT the only possible type signature as highlighted below
+// curriedAdd :: Number -> Number -> Number -> Number
+const curriedAdd = curry( add )
+
+curriedAdd(1)
+// => [Function: curried]
+
```

Comment:
Thinking these next bites of code could be "grouped" with a comment explaining the 🌽-venience of partial application using the `curry` function.